web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -251,6 +256,35 @@ class CVEAudit extends React.Component<Props, State> {
               />
             </div>
           </p>
+          {this.state.auditExecuted === true && (
+            <div>
+              <p>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={
+                    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=" + this.state.cveYear + "-" + this.state.cveNumber
+                  }
+                  data-senna-off="true"
+                >
+                  <IconTag type="external-link" />
+                  {t("MITRE CVE link")}
+                </a>
+                &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Instead of this, I would recommend adding a margin. Perhaps @ncounter knows if we have any utility classes for stuff like this?

Reply:
Unfortunately not, not yet I'd say :smiley: 

I'd suggest to add some like the following: https://github.com/uyuni-project/uyuni/blob/master/branding/css/theme.less#L401
That would also be in line with the way of doing it in the latest version of Bootstrap [Bootstrap 4](https://getbootstrap.com/docs/4.0/utilities/spacing/) which I expect to be a bigger work, but to happen in a certain point in time in the future (I'm looking at you @Etheryte :eyes: :wink: )